Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.3-slim
+
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        build-essential \
+        ca-certificates \
+        curl \
+        default-libmysqlclient-dev \
+        git \
+        libffi-dev \
+        libpq-dev \
+        libssl-dev \
+        mysql-client
+
+
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+    apt-get install -y nodejs && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Does this config work?
I'm not super familiar with Dockerfile configuration (as they're often something you do once, then forget about once they're working), but would have thought you'd need a few more commands, along the lines of (but not necessarily limited to):

```
WORKDIR /app
COPY Gemfile* ./
RUN gem install bundler -v '1.17'
RUN bundle install
COPY . .
```

Reply:
This just sets up the bare ruby environment. The files aren't copied in as compose maps the local directory into the container.